plugins/objectstore/objectstore_api.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2018 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package objectstore
+
+import (
+	"github.com/ligato/networkservicemesh/netmesh/model/netmesh"
+)
+
+// Interface is the interface to a ObjectStore handler plugin
+type Interface interface {
+	ObjectCreated(obj interface{})
+	ObjectDeleted(obj interface{})
+	ObjectUpdated(objOld, objNew interface{})
+	ListNetworkServices() []*netmesh.NetworkService
+	ListNetworkServiceEndpoints() []*netmesh.NetworkServiceEndpoint
```

Comment:
@mestery Since I am still getting up to speed in understanding relations between NS, NSEP and NSCH, these calls just  list all of them. Would be great to have a discussion at one point to finalize this interface.